sdk/monitor/azure-monitor-opentelemetry-exporter/samples/traces/sample_trace.py

```diff
@@ -12,8 +12,8 @@
 from azure.monitor.opentelemetry.exporter import AzureMonitorTraceExporter
 
 
-exporter = AzureMonitorTraceExporter(
-    connection_string = os.environ["APPLICATIONINSIGHTS_CONNECTION_STRING"]
+exporter = AzureMonitorTraceExporter.from_connection_string(
```

Comment:
perhaps add a sample to show that the constructor can be used if there's an env variable set